pkg/kubectl/explain/field_lookup_test.go

```diff
@@ -69,9 +69,9 @@ func TestFindField(t *testing.T) {
 			gotPath = path.GetPath().String()
 		}
 
-		if gotErr != test.err && gotPath != test.expectedPath {
-			t.Errorf("LookupSchemaForField(schema, %v) = (%v, %v), expected (%s, %v)",
-				test.path, gotErr, gotPath, test.expectedPath, test.err)
+		if gotErr != test.err && gotPath == test.expectedPath {
```

Comment:
I don't think this was my intent.

Should it be `if gotErr != test.err || gotPath != test.expectedPath {`. IOW, if any of the two expected value is wrong?

Reply:
Sure, that sounds good. In that case, let's have two if loop - one for each to provide a better error message? I will be updating a second commit for you to take a look. Thanks!